nexus/src/app/silo.rs

```diff
@@ -181,9 +181,9 @@ impl super::Nexus {
                 // external id. The next action depends on the silo's user
                 // provision type.
                 match db_silo.user_provision_type {
-                    // If the user provision type is fixed, do not a new user if
-                    // one does not exist.
-                    db::model::UserProvisionType::Fixed => {
+                    // If the user provision type is ApiOnly, do not a new user
```

Comment:
this doesn't look right - maybe it should be "do not *create* a new user"

Reply:
Fixed in 394c3be9.